api/sixtyfps-cpp/include/sixtyfps.h

```diff
@@ -748,19 +748,19 @@ struct VersionCheckHelper
 /// Enters the main event loop. This is necessary in order to receive
 /// events from the windowing system in order to render to the screen
 /// and react to user input.
-inline void run_event_loop()
+inline int run_event_loop()
 {
     private_api::assert_main_thread();
-    cbindgen_private::sixtyfps_run_event_loop();
+    return cbindgen_private::sixtyfps_run_event_loop();
 }
 
 /// Schedules the main event loop for termination. This function is meant
 /// to be called from callbacks triggered by the UI. After calling the function,
 /// it will return immediately and once control is passed back to the event loop,
 /// the initial call to sixtyfps::run_event_loop() will return.
-inline void quit_event_loop()
+inline void quit_event_loop(int error_code = 0)
```

Comment:
Somewhere further down it's called "exit_code" and I think that's a better name than "error_code", especially when 0 means no error :)